### ISSUE TITLE
Feature request: Easier access to Expand Selections option

### ISSUE THREAD

**Issue by eykamp** [yo1dog/vscode-multi-cursor-search#4]
This extension comes very close to what I'm looking for, and with a simple addition it could be it exactly.  I'd like a way to use the expand selections option without having to go through the advanced menu; basically I want a simple way to do a basic non-regex search and expand the selections to those search results.

All the functionality is already there; it's just a matter of exposing it.

Thanks!

**Comment by yo1dog** [comment]
This is actually already possible. You can assign a keybinding to `yo1dog.multi-cursor-search.regexSearchExpandSelection` like so:

```json
{
  "key": "ctrl+shift+alt+f",
  "command": "yo1dog.multi-cursor-search.plainSearchExpandSelection",
  "when": "editorTextFocus"
}
```

However, it looks like I forgot to add `plainSearchExpandSelection` and `regexSearchExpandSelection` as both activation evens and command contributions. So if you get a "command not found" error, that probably means the extension has not been activated yet. Try doing a normal search first (shift+alt+f) to activate the extension.

I will push a fix today.

**Comment by yo1dog** [comment]
fixed in v2.0.4
b1cf9c5e9cdb2d3b8730b1a7eb7ab0696fc2ce8c
